checks/checks.md

```diff
@@ -86,6 +86,12 @@ This check tries to determine if the project uses a fuzzing system. It currently
 **Remediation steps**
 - Integrate the project with OSS-Fuzz by following the instructions [here](https://google.github.io/oss-fuzz/).
 
+## Packaging 
+
+This check tries to determine if the project is published as a package that other developers can install/download. The check currently only looks for GitHub packaging workflows. There is plan to add support for package manager hubs in the future. 
```

Comment:
Instead of `There is plan` it would be `There is a plan`

Reply:
+1
also add "other package manager hubs" (add other), maybe can add any example if you can think of (X, Y, etc)